components/Registration/Pages/ApplicationSubmitted/ApplicationSubmitted.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Image from 'next/image';
+import styles from './ApplicationSubmitted.module.scss';
+
+const PersonalInfo: React.FC = () => {
+    return (
+        <>
+            <Image
```

Comment:
This image will be part of the navbar, not the application submitted page itself.

Reply:
Correction on this actually, it looks like the hackillinois logo is under the text for the application submitted page on the figma.
![image](https://github.com/user-attachments/assets/c9ae7b4f-5732-4e29-b396-a99d07d1e0b8)